src/main/java/utils/IOUtils.java

```diff
@@ -0,0 +1,21 @@
+package utils;
+
+import org.slf4j.Logger;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class IOUtils {
+    public static String getFileName(String firstLine){
```

Comment:
```suggestion
    public static String getURL(String requestLine){
```

도메인 언어를 사용하는 편이 가독성이 높아집니다.